app/models/archived/dependent_2021.rb

```diff
@@ -202,8 +202,8 @@ def rules_2021
       rules(2021)
     end
 
-    def rules(year)
-      Dependent::Rules.new(birth_date, year, full_time_student_yes?, permanently_totally_disabled_yes?, ssn.present?, qualifying_child_relationship?, qualifying_relative_relationship?, meets_misc_qualifying_relative_requirements_yes?, meets_qc_residence_condition_generic?, meets_qc_claimant_condition?, meets_qc_misc_conditions?)
+    def rules(tax_year)
+      Dependent::Rules.new(self, tax_year)
```

Comment:
I found the implementation that passed in all of the dependent attributes individually a bit difficult to reason about and extend upon. This is a bit of an opinionated change, so I'm open to talking about tradeoffs.